manim/scene/scene_from_video.py

```diff
@@ -1,54 +1,54 @@
-from tqdm import tqdm as show_progress
-import cv2
+# from tqdm import tqdm as show_progress
```

Comment:
Normally I would never merge commented-out files but in the interest of adding Travis support ASAP, I'll turn a blind eye on this.